src/frontend/hooks/server/media.ts

```diff
@@ -30,7 +34,13 @@ export function useCreateBlobMutation(opts: {retry?: number} = {}) {
         },
         // TODO: DraftPhoto type should probably carry MIME type info that feeds this
         // although backend currently only uses first part of path
-        {mimeType: 'image/jpeg'},
+
+        {
+          mimeType: 'image/jpeg',
+          // @ts-expect-error
```

Comment:
opened issue to fix typings here: https://github.com/digidem/mapeo-core-next/issues/719